docs/modules/ROOT/pages/flow.adoc

```diff
@@ -0,0 +1,77 @@
+= Deploy Flow
+
+:description: In this tutorial, you'll deploy a Flow application using Hazelcast Platform Operator for Kubernetes.
+
+{description}
+
+See xref:flow:introduction:index.adoc[Flow] for more detailed information.
+
+== Prerequisites
+
+- Flow requires the Management Center application to run and be configured properly. The Management Center application should be configured to use FLow.
+
+[source,yaml]
+----
+...
+image: docker.io/hazelcast/management-center-flow <1>
+...
+hazelcastClusters:
+- address: flow <2>
+  name: flow <3>
+jvm:
+  args:
+  - "-Dhazelcast.mc.flow.addresses=http://233.252.0.158/" <4>
+  - "-Dhazelcast.mc.internal.flowAddresses=http://flow:9021" <5>
+...
+----
+
+<1> Management Center Flow image.
+<2> The service name of the Flow application. The same as the Flow Custom Resource (CR) name.
+<3> Hazelcast cluster name. Should always be `flow`.
+<4> Flow Ingress public address.
+<5> The internal address of Flow in the format of `http://<service-name>:9021`. The service name is the same as the Flow CR name.
+
+- To connect to the database you need create a `Secret` with the DB credentials:
```

Comment:
The DB have to be Postgre so we should mention in here or at the beginning of the section. WDYT?

Reply:
I thought we didn't mention it on purpose, don't we?

Reply:
Why?

If flow mentioned about it we can give reference to it also

Reply:
Done